### PR TITLE
fix: size options height conversion

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-379] 

### Description

**Before**

https://github.com/artsy/eigen/assets/11945712/e0ebe4ef-9778-4c75-b4cf-e742946b907b

**After**

https://github.com/artsy/eigen/assets/11945712/bea17f84-6f3d-40cb-aad9-e4ba784671a8

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix size options height conversion - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-379]: https://artsyproduct.atlassian.net/browse/ONYX-379?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx [hunk 1]
```diff
@@ -112,6 +111,9 @@ export const SavedSearchAlertForm: React.FC<SavedSearchAlertFormProps> = (props)
             : attributes
         )
 
+        console.log({ unit })
```
Comment (gkartalis) [inline src/app/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx:114]: 👀👀👀

Comment (MounirDhahri) [reply src/app/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx]: removed 😄

Comment (gkartalis) [reply src/app/Scenes/SavedSearchAlert/SavedSearchAlertForm.tsx]: 🧹

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix size options height conversion - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against ef5564f6d5db32d8b8ef7cd230e5ee511e7dc724
</p>

**Review by nickskalkin — APPROVED** [review]

**Pull request merged**